src/length-value.js

```diff
@@ -24,6 +24,19 @@
     'cm', 'mm', 'q', 'in', 'pc', 'pt'
   ];
 
+  LengthValue.cssStringTypeRepresentation = function(type) {
+    if (LengthValue.LengthType.indexOf(type) < 0) {
+      throw new TypeError('Invalid LengthType.');
+    }
+
+    switch (type) {
```

Comment:
This switch statement is a bit excessive, but I thought it would fit with the code style in ComputedStylePropertyMap, plus easy / clear how/where to add more cases.